src/WorkItemMigrator/tests/Migration.Wi-Import.Testss/WitClient/WitClientUtilsTests.cs

```diff
@@ -843,5 +858,67 @@ public void When_calling_save_workitem_with_populated_workitem_Then_workitem_is_
             Assert.That(createdWI.Relations[1].Url, Is.EqualTo($"https://example/workItems/{revision.Links[0].TargetWiId}"));
 
         }
+
+        [Test]
+        public void When_calling_create_sprint_path_from_revision_and_add_to_work_item_with_empty_args_Then_an_exception_is_thrown()
+        {
+            MockedWitClientWrapper witClientWrapper = new MockedWitClientWrapper();
+            WitClientUtils wiUtils = new WitClientUtils(witClientWrapper);
+
+            Assert.That(
+                () => wiUtils.CreateSprintPathFromRevisionAndAddToWorkItem(null, null, null, null, null, null, TreeStructureGroup.Iterations),
+                Throws.InstanceOf<ArgumentException>());
+        }
+
+        [Test]
+        public void When_calling_create_sprint_path_from_revision_and_add_to_work_item_with_valid_args_Then_sprint_is_added_to_work_items()
+        {
+            MockedWitClientWrapper witClientWrapper = new MockedWitClientWrapper();
+            WitClientUtils wiUtils = new WitClientUtils(witClientWrapper);
+
+            // Setup
+
+            WorkItem createdWI = wiUtils.CreateWorkItem("User Story");
+
+            string baseArea = "base-area";
+            string areaValue = "my&area";
+            string baseIteration = "base-iteration";
+            string iterationValue = "my#iteration";
+            string project = "MyProject";
+
+            List<CharReplaceRule> charReplaceRules = new List<CharReplaceRule>();
+            CharReplaceRule crRule1 = new CharReplaceRule();
+            crRule1.Source = "&";
+            crRule1.Target = "-";
+            charReplaceRules.Add(crRule1);
+            CharReplaceRule crRule2 = new CharReplaceRule();
+            crRule2.Source = "#";
+            crRule2.Target = "_";
+            charReplaceRules.Add(crRule2);
+
+            string expectedArea = string.Format("{0}\\{1}\\{2}", project, baseArea, areaValue);
```

Comment:
why not create expected value staticlly as you know what you are expecting and get rid of the over coding ?

Reply:
What do you mean with creating the expected value statically? :) @madkoo

Reply:
So I meant that instead of concatenating and replacing values for a expected value, for simplicity I would use 
expectedArea = "MyProject\\base-area\\my-area"   it would be less code, its up to you. 
Also you could use string interpolation instead string.format, but in the end same result